django/library/jinja2/library/codebases/releases/edit.jinja

```diff
@@ -1,9 +1,13 @@
 {% extends "base.jinja" %}
+{% from "common.jinja" import spam_detected_alert %}
 
 {% block introduction %}
 <h1><i class='fas fa-cloud-upload-alt'></i> Publish your computational model</h1>
 {% endblock %}
 {% block content %}
+{% if release.codebase.is_marked_spam %}
+    {{ spam_detected_alert() }}
```

Comment:
Assume this should be `alert_if_spam(true)` ? and prevent release editing if marked as spam?